lib/controllers/ion_controller.dart

```diff
@@ -5,15 +5,20 @@ import 'package:shared_preferences/shared_preferences.dart';
 
 class IonController extends GetxController {
   SharedPreferences? _prefs;
-  IonConnector? _ion;
+  IonAppBiz? _ion;
   late String _sid;
   late String _name;
   final String _uid = Uuid().v4();
-  IonConnector? get ion => _ion;
+  Client? clientSFU;
```

Comment:
You can use IonSDKSFU instead of Client + GRPCWebSignal. The new SDK uses IonBaseConnector to construct IonService. The advantage of this is that we can share metadata, such as tokens, and node information through IonBaseConnector.
https://github.com/pion/ion-sdk-flutter/blob/master/lib/src/connector/sfu.dart#L15

look like this.
```dart
    _sfu = ion.IonSDKSFU(_connector!);

    _sfu?.onspeaker = (Map<String, dynamic> list) {
      print('onspeaker: $list');
    };

    _sfu?.ontrack = (track, ion.RemoteStream remoteStream) async {
      if (track.kind == 'video') {
        print('onTrack: remote stream => ${remoteStream.id}');
      }
    };

```

Reply:
Thank you! I modified the code to use IonSDKSFU instead of instead of Client + GRPCWebSignal.